app/models/manageiq/providers/storage_manager.rb

```diff
@@ -12,5 +12,13 @@ class StorageManager < ManageIQ::Providers::BaseManager
                :foreign_key => :parent_ems_id,
                :class_name  => "ManageIQ::Providers::BaseManager",
                :autosave    => true
+
+    def cinder_service_available?
+      raise NotImplementedError, _("cinder_service_available? must be implemented in a subclass")
+    end
+
+    def swift_service_available?
+      raise NotImplementedError, _("swift_service_available? must be implemented in a subclass")
+    end
```

Comment:
@hsong-rh we are working on block and object storage manager for AWS where these two methods are not going to be used. UI works and links storage with cloud manager. I think it should be safe to remove this from the base storage manager.